# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296853130 28800
#      Fri Feb 04 12:58:50 2011 -0800
# Branch release-3-4-x
# Node ID ab838df8de9049173a5d1d9e9ad3cf8fbbaa8aa9
# Parent  1adc6b3d943d24aaf6254dae099d469ea84a99c9
Undo accidental checkin.

diff --git a/liboctave/ChangeLog b/liboctave/ChangeLog
--- a/liboctave/ChangeLog
+++ b/liboctave/ChangeLog
@@ -1,8 +1,12 @@
+2010-02-04  Rik  <octave@nomad.inbox5.com>
+
+	* Makefile.am: Undo accidental checkin
+
 2011-02-03  John W. Eaton  <jwe@octave.org>
 
 	* cmd-hist.cc (gnu_history::do_process_histcontrol):
 	Rename from command_history::do_process_histcontrol.
 	(gnu_history::do_histcontrol): Rename from
 	command_history::do_histcontrol.
 	* cmd-hist.h (command_history::do_histcontrol): Provide dummy version.
 	* cmd-hist.cc (command_history::do_process_histcontrol): Likewise.
diff --git a/liboctave/Makefile.am b/liboctave/Makefile.am
--- a/liboctave/Makefile.am
+++ b/liboctave/Makefile.am
@@ -260,26 +260,23 @@ OTHER_INC = \
   intNDArray.cc \
   kpse.cc \
   mx-inlines.cc
 
 include vx-op-inc.mk
 include mx-op-inc.mk
 include smx-op-inc.mk
 
-#$(srcdir)/vx-op-inc.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/vx-ops
-vx-op-inc.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/vx-ops
+$(srcdir)/vx-op-inc.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/vx-ops
 	$(srcdir)/config-ops.sh $(top_srcdir) vx inc
 
-#$(srcdir)/mx-op-inc.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/mx-ops
-mx-op-inc.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/mx-ops
+$(srcdir)/mx-op-inc.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/mx-ops
 	$(srcdir)/config-ops.sh $(top_srcdir) mx inc
 
-#$(srcdir)/smx-op-inc.mk: $(srcdir)/config-ops.sh $(srcdir)/sparse-mk-ops.awk $(srcdir)/sparse-mx-ops
-smx-op-inc.mk: $(srcdir)/config-ops.sh $(srcdir)/sparse-mk-ops.awk $(srcdir)/sparse-mx-ops
+$(srcdir)/smx-op-inc.mk: $(srcdir)/config-ops.sh $(srcdir)/sparse-mk-ops.awk $(srcdir)/sparse-mx-ops
 	$(srcdir)/config-ops.sh $(top_srcdir) smx inc
 
 BUILT_INCS = \
   mx-ops.h \
   $(OPT_INC) \
   $(MX_OP_INC) \
   $(VX_OP_INC) \
   $(SMX_OP_INC)
@@ -460,26 +457,23 @@ LIBOCTAVE_CXX_SOURCES = \
   str-vec.cc \
   $(TI_SRC) \
   $(MATRIX_SRC)
 
 include vx-op-src.mk
 include mx-op-src.mk
 include smx-op-src.mk
 
-#$(srcdir)/vx-op-src.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/vx-ops
-vx-op-src.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/vx-ops
+$(srcdir)/vx-op-src.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/vx-ops
 	$(srcdir)/config-ops.sh $(top_srcdir) vx src
 
-#$(srcdir)/mx-op-src.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/mx-ops
-mx-op-src.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/mx-ops
+$(srcdir)/mx-op-src.mk: $(srcdir)/config-ops.sh $(srcdir)/mk-ops.awk $(srcdir)/mx-ops
 	$(srcdir)/config-ops.sh $(top_srcdir) mx src
 
-#$(srcdir)/smx-op-src.mk: $(srcdir)/config-ops.sh $(srcdir)/sparse-mk-ops.awk $(srcdir)/sparse-mx-ops
-smx-op-src.mk: $(srcdir)/config-ops.sh $(srcdir)/sparse-mk-ops.awk $(srcdir)/sparse-mx-ops
+$(srcdir)/smx-op-src.mk: $(srcdir)/config-ops.sh $(srcdir)/sparse-mk-ops.awk $(srcdir)/sparse-mx-ops
 	$(srcdir)/config-ops.sh $(top_srcdir) smx src
 
 BUILT_LIBOCTAVE_CXX_SOURCES = \
   $(MX_OP_SRC) \
   $(VX_OP_SRC) \
   $(SMX_OP_SRC)
 
 LIBOCTAVE_C_SOURCES = \
